_posts/2016-12-05-integrated-shrinking.md

```diff
@@ -146,7 +146,7 @@ Haskell](https://mail.haskell.org/pipermail/libraries/2013-November/021674.html)
 and [Jack](https://github.com/ambiata/disorder.hs/tree/master/disorder-jack) is
 an alternative property based system for Haskell that does this - but you
 feel the pain much more quickly in dynamic languages because the typical
-workaround for this problem in Haskell is to define a newtype, which lets you
+workaround for this problem in Haskell is to define a new type, which lets you
```

Comment:
This is actually not a spelling mistake, but a Haskell specific term. newtype is the keyword used for defining lightweight wrapper types that do nothing but wrap a single value of another type.